hollow/src/main/java/com/netflix/hollow/core/memory/HollowUnsafeHandle.java

```diff
@@ -38,10 +44,102 @@
             log.log(Level.SEVERE, "Unsafe access failed", e);
         }
         unsafe = u;
+        singleton = new HollowUnsafeHandle();
+    }
+
+    public static HollowUnsafeHandle getUnsafe() {
+        return singleton;
```

Comment:
We could return an implementation of `HollowUnsafeHandle` that always does unaligned, or an instance that always does aligned. Then we don't incur the cost of branching in the read path.

Reply:
I'm fairly certain that, since the guard is a final, the JVM will optimize out the branch not-taken when it compiles this code to native, but I can swap to using two implementations to be more certain.

Reply:
Ah I didn't consider that.